deepmd/model/ener.py

```diff
@@ -454,3 +465,38 @@ def natoms_not_match(self, force, natoms, atype):
         ghost_force = tf.concat([ghost_force_real, ghost_force_mag], axis=1)
         force = tf.concat([loc_force, ghost_force], axis=1)
         return force
+
+    def change_energy_bias(
+        self,
+        data: DeepmdDataSystem,
+        frozen_model: str,
+        origin_type_map: list,
+        full_type_map: str,
+        bias_shift: str = "delta",
+    ) -> None:
+        """Change the energy bias according to the input data and the pretrained model.
+
+        Parameters
+        ----------
+        data : DeepmdDataSystem
+            The training data.
+        frozen_model : str
+            The path file of frozen model.
+        origin_type_map : list
+            The original type_map in dataset, they are targets to change the energy bias.
+        full_type_map : str
+            The full type_map in pretrained model
+        bias_shift : str
+            The mode for changing energy bias : ['delta', 'statistic']
+            'delta' : perform predictions on energies of target dataset,
+                    and do least sqaure on the errors to obtain the target shift as bias.
+            'statistic' : directly use the statistic energy bias in the target dataset.
+        """
+        self.fitting.change_energy_bias(
+            data,
+            frozen_model,
+            origin_type_map,
+            full_type_map,
+            bias_shift,
+            self.data_stat_nbatch,
```

Comment:
While `data_stat_nbatch` is suitable for data stating, maybe it's sometimes too large for `change_energy_bias`?(since this will triger GPU computation)

Reply:
I see. It's a typo.